_health-care/_js/_components/_health-care-app.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Nav } from './_hello-world.jsx';
```

Comment:
Yup! 


Though funny enough...I think I was actually copying your pattern....at least I recall going "huh, Anne did a default export in her PR. That's a good idea."